translations/base-fr.yaml

```
@@ -30,59 +30,59 @@ steamPage:
+        Acheter le jeu vous donne accès à la version complète qui a des fonctionnalitées additionnelles et vous recevrez aussi un accès à des fonctionnalitées fraîchement développées.
 
-        [b]Standalone Advantages[/b]
+        [b]Avantages de la version complète[/b]
 
         [list]
-            [*] Dark Mode
-            [*] Unlimited Waypoints
-            [*] Unlimited Savegames
-            [*] Additional settings
-            [*] Coming soon: Wires & Energy! Aiming for (roughly) end of July 2020.
-            [*] Coming soon: More Levels
-            [*] Allows me to further develop shapez.io ❤️
+            [*] Mode sombre
+            [*] Marqueurs illimités
+            [*] Nombre illimité de parties
+            [*] Paramètres additionnaux
+            [*] Arrivage imminent: Câbles et énergie! Visé pour (environ) fin Juillet 2020.
+            [*] Arrivage imminent: Plus de niveaux
+            [*] Aidez moi à continuer de développer shapez.io ❤️
         [/list]
 
-        [b]Future Updates[/b]
+        [b]Mises à jour futures[/b]
 
-        I am updating the game very often and trying to push an update at least every week!
+        Je met à jour le jeu très souvent et j'essaye de sortir une mise à jour au moins toutes les semaines!
 
         [list]
-            [*] Different maps and challenges (e.g. maps with obstacles)
-            [*] Puzzles (Deliver the requested shape with a restricted area / set of buildings)
-            [*] A story mode where buildings have a cost
-            [*] Configurable map generator (Configure resource/shape size/density, seed and more)
-            [*] Additional types of shapes
-            [*] Performance improvements (The game already runs pretty well!)
-            [*] And much more!
+            [*] Différentes cartes at challenges (ex: carte avec obstacles)
+            [*] Puzzles (Délivrer la forme requise avec une zone limitée/jeu de batîments
+            [*] Un mode histoire où les batîments ont un coût
+            [*] Générateur de carte configurable (configuration des ressources/formes taille/densitée, graine et plus)
+            [*] Types de formes additionnelles
+            [*] Amélioration des performances (Le jeu tourne déjà plutot bien!)
+            [*] Et plus encore!
         [/list]
 
-        [b]This game is open source![/b]
+        [b]Ce jeu est open source![/b]
 
-        Anybody can contribute, I'm actively involved in the community and attempt to review all suggestions and take feedback into consideration where possible.
-        Be sure to check out my trello board for the full roadmap!
+        N'importe qui peut contribuer, je suis activement impliqué dans la communauté et j'essaye de répondre à toutes les suggestions et prendre en compte votre retour d'expérience quand je le peux.
+        Soyez sûr de jeter un coup d'œil à mon tableau Trello pour toute la feuille de route!
```

```suggestion
        Soyez sûr de jeter un coup d'œil à mon tableau de bord Trello pour le suivi du projet et la planification du développement
```

"roadmap", I prefere "planification du développement"